package.json

```
@@ -4,32 +4,32 @@
   "private": true,
   "homepage": "./",
   "dependencies": {
-    "@excalidraw/excalidraw": "0.12.0",
-    "browser-fs-access": "0.31.0",
-    "excalidraw-animate": "0.5.0",
+    "@excalidraw/excalidraw": "0.15.2",
+    "browser-fs-access": "^0.33.1",
+    "excalidraw-animate": "0.6.0",
     "gif.js": "^0.2.0",
     "lodash": "^4.17.21",
-    "nanoid": "3.3.4",
+    "nanoid": "4.0.2",
```

I think it's easier to use the old version for now. We could eliminate nanoid and look for an alternative.
```suggestion
    "nanoid": "3.3.4",
```

---

nanoid v4 drop support for CommonJS. looks like that's what caused the error